docs/getting-started.md

```diff
@@ -2,12 +2,21 @@
 
 *If you've previously installed gulp globally, run `npm rm --global gulp` before following these instructions.*
 
+*Make sure that you've installed Node and NPM before attempting to install gulp.*
+
 #### Install the `gulp` command
 
 ```sh
 npm install --global gulp-cli
 ```
 
+#### Create a package.json
+If you don't have a package.json, create one. If you need help, run an npm init which will walk you through giving it a name, version, and description.
```

Comment:
wrap `package.json` in code ticks.  Same with `npm init`; should probably say "etc" of things it walks through.